habitat_baselines/rl/ddppo/policy/running_mean_and_var.py

```diff
@@ -62,7 +63,7 @@ def forward(self, x: Tensor) -> Tensor:
 
             self._count += new_count
 
-        stdev = torch.sqrt(
+        inv_stdev = torch.rsqrt(
             torch.max(self._var, torch.full_like(self._var, 1e-2))
         )
-        return (x - self._mean) / stdev
+        return torch.addcmul(-self._mean * inv_stdev, x, inv_stdev)
```

Comment:
It doesn't look cleaner with rsqrt and addcmul.

Reply:
Agreed, but this is faster and and gives extra stability for fp16 since it is a fused op.  I can move this to that PR if you'd prefer

Reply:
Really, it's faster than rsqrt? I thought the hwole point or rsqrt is that it uses dedicated instructions which are faster than sqrt + divide?

Reply:
> Really, it's faster than rsqrt?
@erikwijmans said that rsqrt and addcmul are faster and and gives extra stability for fp16.
I am totally ok to have it in this PR, just was curious.

Reply:
Ah, I forgot we were dealing with FP16, yeah rsqrt is going to have stability issues with that.

Reply:
This PR is swapping to using rsqrt because it is more stable (actually that is still done in fp32, but the addcmul is done in fp16 in the fp16 branch)

Reply:
I double checked, rsqrt is going to be more stable in fp16 because sqrt is implemented as 1.0 / rsqrt(x) unless Nvidia added a hardware instruction for sqrt recently.